book/src/k8s-test-runner/k8s-test-runner.md

```diff
@@ -130,6 +140,11 @@ Then:
         WASP_LOG_LEVEL = "info"
         TEST_LOG_LEVEL = "info"
         MERCURY_TEST_LOG_LEVEL = "info"
+        [metadata.labels]
+        "chain.link/component" = "k8s-test-runner"
+        "chain.link/product" = "data-streamsv0.3"
+        "chain.link/team" = "Data Streams"
+        "chain.link/cost-center" = "data-streamsv0.3-load-test"
```

Comment:
cost-center should be something like tt-load-test or some product/framework name (owned by test-tooling team) that identifies who to reach out to for inefficiencies in the framework. for example if all aws infrastructure costs go up by 15% month over month for multiple months and this is a infra-platform framework then we would label it to an identifier that maps to infra platform

Reply:
but is this one incorrect? `<product>-<test-type>-test`. In this very case this test is not owned by the TT, we just wrote the k8s runner that it uses.